prac_08/taxi_simulator.py

```diff
@@ -0,0 +1,50 @@
+"""
+CP1404 | Prac_08 | Dannielle Jones
+
+"""
+from prac_08.taxi import Taxi
+from prac_08.silver_service_taxi import SilverServiceTaxi
+MENU = "Q) uit, C) hoose taxi, D) rive \n>>> "
+
+
+def main():
+    taxis = [Taxi("Prius", 100), SilverServiceTaxi("Limo", 100, 2), SilverServiceTaxi("Hummer", 200, 4)]
+    current_taxi = None
+    total_cost = 0
+    print("Let's Drive!")
+    choice = input(MENU).upper()
+    while choice != "Q":
+        if choice == "C":
+            print("Taxis available:")
+            display_taxis(taxis)
+            try:
+                taxi_choice = int(input("Choose taxi: "))
```

Comment:
This line should be outside the suite

Reply:
or you could have an extra `except ValueError` catch.